src/index.js

```diff
@@ -1,21 +1,22 @@
-import express from 'express'
 import mongoose from 'mongoose';
-import router from'./routes/router';
-import authRoutes from'./routes/auth';
-import passport from 'passport';
-
-
-mongoose.set('strictQuery', false)
-mongoose.connect("mongodb://localhost:27017/serverDB", { useNewUrlParser: true }).then(() => {
-    const app = express();
-    app.use(express.json())
-    app.use("/api",router);
-    app.use('/user', authRoutes)
-   
+import express from 'express'
+import router from './routes/router';
+import authRoutes from './routes/auth';
+const app = express();
+app.use(express.json({limit: '500mb'}))
+// app.use(express.urlencoded({limit:'500mb'}))
 
+try {
+    mongoose.set('strictQuery', false)
+    mongoose.connect("mongodb://localhost:27017/serverDB", { useNewUrlParser: true });
```

Comment:
Don't expose your db URL. Use env variables.

Reply:
i've completed that i've hidden it in the env variable